dozer/util.py

```diff
@@ -19,3 +19,22 @@ def asbool(obj):
         else:
             raise ValueError("String is not true/false: %r" % obj)
     return bool(obj)
+
+
+def monotonicity(objs):
+    # Monotonicity is a measurement of value increment over time
+    # Large monotonicity indicates that number of objects
+    # has been increased for a while, where leakage is likely to happen
+
+    if len(objs) == 0:
+        return 0
+    der = [0 for _ in range(len(objs))]
+    for i in range(len(objs) - 1):
+        der[i+1] = objs[i+1] - objs[i]
+    inc_cnt = len(list(filter(lambda x: x>0, der))) + 0.0
+    dec_cnt = len(list(filter(lambda x: x<=0, der))) + 0.0
+    return inc_cnt / (inc_cnt + dec_cnt)
```

Comment:
This seems expensive.  Surely we could compute inc_cnt and dec_cnt by doing one loop through objs?  Something line this ought to work:

```python
inc_cnt = 0
dec_cnt = 1
for i in range(len(objs) - 1):
    if objs[i+1] > objs[i]:
        inc_cnt += 1
    else:
        dec_cnt += 1
```